app/assets/stylesheets/app/card.scss

```diff
@@ -159,4 +160,86 @@
       }
     }
   }
+  &-pr {
+    @extend .card;
+    border-radius: 10px;
+    margin-top: 20px;
+    margin-bottom: 1px;
+    margin-right: 5px;
+    margin-left: 0px;
+    display: flex; 
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected newline after ";" in a multi-line declaration block   declaration-block-semicolon-newline-after